webpack.config.js

```diff
@@ -78,6 +78,10 @@ module.exports = function(env) {
 		},
 		resolve: {
 			extensions: ['.js', '.ts', '.json'],
+			alias: {
+				// Use the es2018 build instead of the default es2015 build
+				'pinejs-client-core': 'pinejs-client-core/es2018',
```

Comment:
As the comment says - and es2018 is nice as it lets us use a build with native async/await rather than transpiled